graylog2-web-interface/src/components/nodes/NodesActions.jsx

```diff
@@ -38,6 +39,7 @@ const NodesActions = React.createClass({
     }
   },
   render() {
+    const apiBrowserURI = new URI(this.props.node.transport_address).directory('api-browser');
```

Comment:
While certainly not nice, this would work:
```
> URI("http://example.com/foo" + "/" + "bar").toString()
"http://example.com/foo/bar"
> URI("http://example.com/foo/" + "/" + "bar").toString()
"http://example.com/foo//bar"
```
Or with the given code snippet:
```javascript
    const apiBrowserURI = new URI(this.props.node.transport_address + '/api-browser');
```
So in the end, this isn't *much* better than simply concatenating those strings but at least the result URI is normalized.